meta-genivi-dev/meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/bluetoothpairingapp_git.bb

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2017 Viktor Sjölind <viktor.sjolind@pelagicore.com>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+DESCRIPTION = "Application used to pair bluetooth devices"
+LICENSE = "MPL-2.0"
+
+DEPENDS = "qtbase qtdeclarative"
+
+RDEPENDS_${PN} = "blueconnect"
+
+inherit qmake5
+
+SRC_URI = "git://github.com/dunderroffe/bluetoothPairingApp;branch=master"
+SRCREV = "ad09140f7fb2ce3056a3827729bc833fbb485e8f"
+
+APP = "com.genivi.gdp.bluetoothPairingApp"
+EXE = "bluetoothPairingApp"
+DESKTOP_FILE = "${APP}.desktop"
+SRC_URI_append = " file://${DESKTOP_FILE}"
+
+S = "${WORKDIR}/git"
+
+
```

Comment:
This is specific to how Yocto builds the application hence it needs to be here not in the application repository. Furthermore, I am not sure it would help to put it in the qmake file.